newsfragments/194.feature.rst

```diff
@@ -0,0 +1 @@
+Added `humanize_wei`` utility to convert common values to more readable units.
```

Comment:
nit: this won't pick up proper formatting

```suggestion
Added ``humanize_wei`` utility to convert common values to more readable units.
```

Reply:
Whoops, thanks for catching!